code/UI/Crosshair.cs

```diff
@@ -10,6 +12,15 @@ public Crosshair()
             StyleSheet.Load("ui/crosshair.scss");
             selectCircle = AddChild<SelectCircle>();
         }
+        [Event("buildinput")]
+        public void ProcessClientInput(InputBuilder input)
+        {
+            if (Local.Pawn is not MainPlayer player)
+                return;
+
+            if (input.Pressed(InputButton.Attack2)) player.Holding.UseOn(player);
```

Comment:
It is not crosshair responsibility, we should do a full UI rework anyway

Reply:
No need a rework ! 
We can make a command for use the holding ? It's better ?

Reply:
Yep a command would be fine but is it logical to have the "Use" part in the crosshair component ? For me, crosshair is not the component responsible for item usage

Reply:
Yeah ok you convince me. Bu do we have to put the inputs outside ui ?

Reply:
maybe we can make a inputHandler

Reply:
Related to #219